CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ All notable changes to this project will be documented in this file. The format
   - `VERSION` was incremented for prior releases to be notified of the v13.0.1 patch release ([#3676](https://github.com/docker-mailserver/docker-mailserver/pull/3676))
   - `VERSION` is no longer included in the image ([#3711](https://github.com/docker-mailserver/docker-mailserver/pull/3711))
   - Update-check: fix 'read' exit status ([#3688](https://github.com/docker-mailserver/docker-mailserver/pull/3688))
+  - Fix error, when ENABLE_QUOTAS=0 was set ([#3715](https://github.com/docker-mailserver/docker-mailserver/pull/3715))
```

Comment:
There wasn't an error in prior releases though? 😆 

```suggestion
  - `ENABLE_QUOTAS=0` no longer attempts to remove config that was never there ([#3715](https://github.com/docker-mailserver/docker-mailserver/pull/3715))
```

Reply:
No visible error (message) at least 👍 But a not working statement I would call erroneous. However, your wording is better 🎉